data/src/main/java/com/antosik/benchproject/data/common/modules/MoviesModule.kt

```diff
@@ -32,9 +35,18 @@ object MoviesModule {
         fun provideMoviesApi(retrofit: Retrofit): MoviesApi = retrofit.create()
 
         @Provides
-        fun providePopularMoviesRepository(apiService: MoviesApi): PopularMoviesRepository = PopularMoviesDataRepository(apiService)
+        fun providePopularMoviesRepository(
+            apiService: MoviesApi
+        ): PopularMoviesRepository = PopularMoviesDataRepository(apiService)
 
         @Provides
-        fun provideMovieDetailsRepository(apiService: MoviesApi): MovieDetailsRepository = MovieDetailsDataRepository(apiService)
+        fun provideMovieDetailsRepository(apiService: MoviesApi): MovieDetailsRepository =
+            MovieDetailsDataRepository(apiService)
+
+        @Provides
+        fun provideMovieDatabase(app: Application): MovieDatabase =
+            Room.databaseBuilder(
+                app, MovieDatabase::class.java, "movie_database"
```

Comment:
Minor: New line do not improve readability in this case in my opinion :D

Reply:
I agree with it, therefore I will write it in one line :D